src/widget/stock.rs

```diff
@@ -704,7 +704,7 @@ impl CachableWidget<StockState> for StockWidget {
                 ]),
                 Spans::default(),
                 Spans::from(vec![
-                    Span::styled("v: ", style()),
+                    Span::styled("Volume: ", style()),
```

Comment:
Any reason we use Volume here instead of V or Vol? I have no preference, just curious. I think it looks good.

Reply:
No particular reason. I guess it should be V instead of Volume but I wasn't sure if V would be explicit enough as it could possibly be confused with other statistics.

Reply:
Agreed, I'm good leaving it this way. We have the space.